components/dashboard/src/teams/OrgSettingsPage.tsx

```diff
@@ -4,39 +4,42 @@
  * See License.AGPL.txt in the project root for license information.
  */
 
+import { Team } from "@gitpod/gitpod-protocol";
+import { BillingMode } from "@gitpod/gitpod-protocol/lib/billing-mode";
 import { useMemo } from "react";
 import { Redirect } from "react-router";
 import Header from "../components/Header";
 import { SpinnerLoader } from "../components/Loader";
 import { PageWithSubMenu } from "../components/PageWithSubMenu";
 import { useFeatureFlags } from "../contexts/FeatureFlagContext";
+import { useOrgBillingMode } from "../data/billing-mode/org-billing-mode-query";
 import { useCurrentOrg } from "../data/organizations/orgs-query";
-import { getTeamSettingsMenu } from "./TeamSettings";
 
 export interface OrgSettingsPageProps {
     children: React.ReactNode;
 }
 
 export function OrgSettingsPage({ children }: OrgSettingsPageProps) {
     const org = useCurrentOrg();
+    const orgBillingMode = useOrgBillingMode();
```

Comment:
Just wondered if there are any (potential) downsides to splitting this kind of information/API wise: I imagine this way `org` and `orgBillingMode` could be out of sync for some render cycles, right? :thinking:

Reply:
(just a curious question, I don't think we should do anytime about it!)